src/tests/prefer-block-comments-for-declarations.test.ts

```
@@ -13,6 +13,38 @@ ruleTesterTs.run('prefer-block-comments-for-declarations', preferBlockCommentsFo
     `/** API base URL */
     const BASE_URL = 'https://api.example.com';`,
 
+    // ESLint directive comment should be ignored - disable-next-line
```

ESLint

[error] Insert `··`

(prettier/prettier)